src/test/java/com/xceptance/neodymium/testclasses/context/IsSiteTests.java

```diff
@@ -0,0 +1,107 @@
+package com.xceptance.neodymium.testclasses.context;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.xceptance.neodymium.NeodymiumRunner;
+import com.xceptance.neodymium.util.Neodymium;
+
+@RunWith(NeodymiumRunner.class)
+public class IsSiteTests
+{
+    @Test
+    public void happyPath() throws Exception
+    {
+        Neodymium.configuration().setProperty("neodymium.url.site", "USA");
+        String expectedSite = "USA";
+        Assert.assertTrue(Neodymium.isSite(expectedSite));
+    }
+
+    @Test
+    public void siteNull() throws Exception
+    {
+        Neodymium.configuration().setProperty("neodymium.url.site", null);
+        String expectedSite = "USA";
+        Assert.assertFalse(Neodymium.isSite(expectedSite));
+    }
+
+    @Test
+    public void expectedSiteNull() throws Exception
+    {
+        Neodymium.configuration().setProperty("neodymium.url.site", "USA");
+        String expectedSite = null;
+        Assert.assertFalse(Neodymium.isSite(expectedSite));
+    }
+
+    @Test
+    public void noMatch() throws Exception
+    {
+        Neodymium.configuration().setProperty("neodymium.url.site", "USA");
+        String expectedSite = "DE";
+        Assert.assertFalse(Neodymium.isSite(expectedSite));
+    }
+
+    @Test
+    public void noMatchEmptyString() throws Exception
+    {
+        Neodymium.configuration().setProperty("neodymium.url.site", "");
+        String expectedSite = "DE";
+        Assert.assertFalse(Neodymium.isSite(expectedSite));
+    }
+
+    @Test
+    public void varcharMatch() throws Exception
+    {
+        Neodymium.configuration().setProperty("neodymium.url.site", "USA");
+        String[] expectedSites =
+            {
+                "USA", "DE"
+            };
+        Assert.assertTrue(Neodymium.isSite(expectedSites));
+    }
+
+    @Test
+    public void reverseVarcharMatch() throws Exception
+    {
+        Neodymium.configuration().setProperty("neodymium.url.site", "USA");
+        String[] expectedSites =
+            {
+                "DE", "USA"
+            };
+        Assert.assertTrue(Neodymium.isSite(expectedSites));
+    }
+
+    @Test
+    public void noMatchVarchar() throws Exception
+    {
+        Neodymium.configuration().setProperty("neodymium.url.site", "USA");
+        String[] expectedSites =
+            {
+                "DE", "UK"
+            };
+        Assert.assertFalse(Neodymium.isSite(expectedSites));
+    }
+
+    @Test
+    public void nullVarchar() throws Exception
+    {
+        Neodymium.configuration().setProperty("neodymium.url.site", null);
+        String[] expectedSites =
+            {
+                "DE", "UK"
+            };
+        Assert.assertFalse(Neodymium.isSite(expectedSites));
+    }
+
+    @Test
+    public void ExpectedNullVarchar() throws Exception
```

Comment:
please use the standard Java naming scheme >> no capital letters for names of functions